practice-two/src/styles/sections/team.css

```diff
@@ -56,6 +56,18 @@
     margin-bottom: 23px;
   }
 
+  .craig-todd { order: 1; }
+
+  .jennie-graham { order: 3; }
+
+  .chase-meyer { order: 5; }
+
+  .vera-hopkins { order: 2; }
+
+  .eunice-wong { order: 4; }
+
+  .johnny-vargas { order: 6; }
```

Comment:
Code style. Should be 

```.johnny-vargas { 
    order: 6; 
   }
   ````
   
   Don't use the username to define class name